### ISSUE TITLE
BlinkCard/BlinkCard.h file not found

### ISSUE THREAD

**Issue by pku-arun** [BlinkID/blinkid-react-native#180]
"blinkid-react-native": "^5.12.0",
"@microblink/blinkcard-react-native": "^2.4.0",
"react": "16.13.1",
"react-native": "0.63.4",
npm: '6.13.4',
Xcode:12.5.1

Showing Recent Errors Only
node_modules/blinkid-react-native/src/ios/MicroblinkModule/MicroblinkModule/Recognizers/Wrappers/MBVisaRecognizerWrapper.m:1:

Build fails with above error

**Comment by mparadina** [comment]
Hi @pku-arun 

Can you please tell me when does the issue occur?

When you try to add one of the SDKs into your project, or when you try to run and build the project?
Also to add, do you get any additional error messages apart from this one?

Regards,
Milan

**Comment by pku-arun** [comment]
Hi @mparadina ,

Initially, there was only a blinkid-react-native library. Recently we planned to add  scanning card as well and we added blinkcard library as well and the issue started to pop up.  I can see only this line of error , nothing else which was occuring when trying to run and build the project

**Comment by mparadina** [comment]
Hi @pku-arun 

Thank you for the feedback.

Just to double-check, do you have Git LFS installed?

Also to ask, did you maybe get a chance to run the sample applications (both BlinkID and BlinkCard) on your side to see if you get any exceptions?

Regards,
Milan

**Comment by pku-arun** [comment]
Hi @mparadina 

I just created a fresh project and added these libraries. The issue still persist.  Easy to replicate

npx react-native init TestProject
npm install --save blinkid-react-native @microblink/blinkcard-react-native
cd ios && pod install
xed ios -> run/build project

you can replicate the issue

**Comment by mparadina** [comment]
Hi @pku-arun 

We've managed to reproduce the issue with the mentioned instructions.

Once we find out where the source of the issue is located, we will get back to you here.
In the meantime, let us know if you have any additional questions.

Regards,
Milan

**Comment by pku-arun** [comment]
@mparadina 

No. Request you to close the issue as soon as possible as our app release is blocked due to this issue. Thanks a lot for looking into the issue.

Regards

**Comment by pku-arun** [comment]
@mparadina 

Any possible solution for this?

Regards

**Comment by mparadina** [comment]
Hi @pku-arun 

We are currently fixing the issue and I will update you here once I get more information from the developers regarding the fix.
Let me know if you have any additional questions.

Regards,
Milan

**Comment by mparadina** [comment]
Hi @pku-arun 

We've released BlinkCard 2.4.1 with the fix for the issue which prevented the BlinkCard SDK to be combined with our other SDKs:
https://github.com/blinkcard/blinkcard-react-native/releases/tag/v2.4.1

Please test it out and let us know if it fixed the issue.

Regards,
Milan

**Comment by mparadina** [comment]
Hi @pku-arun 

Just a quick follow-up here, did you manage to successfully build the project with the 2.4.1 version?

Regards,
Milan
